app/javascript/components/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Greeting from './Greeting.js'
+const App = () => {
+  return (
+    <div>
+        <Greeting />
```

Comment:
Well done on your progress so far! I noticed that while the random greeting message is displaying, there are still some changes that need to be made. 

- [ ] At present, the Greeting component has been added directly to the App component. However, in line with the project requirements, the App component should have a router that directs to the Greeting component. I will recommend you use the react-router for that.

Example:
```suggestion
        <Route path="/" element={<Greeting />} />
```